test/e2e/data/e2e_conf.yaml

```diff
@@ -98,7 +98,7 @@ providers:
           - sourcePath: "./infrastructure-aws/withoutclusterclass/generated/cluster-template-efs-support.yaml"
           - sourcePath: "./infrastructure-aws/withoutclusterclass/generated/cluster-template-external-csi.yaml"
           - sourcePath: "./infrastructure-aws/withoutclusterclass/generated/cluster-template-csimigration-off.yaml"
-          - sourcePath: "./infrastructure-aws/withoutclusterclass/generated/cluster-template-external-cloud-provider.yaml"
+          - sourcePath: "./infrastructure-aws/withoutclusterclass/generated/cluster-template-intree-cloud-provider.yaml"
```

Comment:
Any reason to change it to intree. This test uses external CSI and external CCM.